assets/css/_fantasy_teams.scss

```diff
@@ -55,17 +65,9 @@
   }
 }
 
-.fantasy-team-table {
-
-  th.points,
-  td.points {
-    text-align: right;
-  }
 
-  th.rank,
-  td.rank {
-    text-align: left;
-  }
+td.points-0 {
+  color: $medium-gray
```

Comment:
Declaration should be terminated by a semicolon

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#trailingsemicolon)